jobs/opensearch/templates/bin/ctl

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+export KEYSTORE="/var/vcap/packages/openjdk-17/lib/security/cacerts"
+export JOB_NAME=opensearch
+export JOB_DIR=/var/vcap/jobs/$JOB_NAME
+STOREPASS="changeit"
```

Comment:
This value should not be hardcoded. It should be a job spec variable and be passed in from BOSH

We should just add a BOSH password to the deployment manifest and pass it in